requests/test/src/requests/RequestTests.scala

```diff
@@ -191,16 +191,15 @@ object RequestTests extends TestSuite{
         compress = requests.Compress.Gzip,
         data = new RequestBlob.ByteSourceRequestBlob("I am cow")
       )
-      assert(read(new String(res2.bytes))("data").toString ==
-        """"data:application/octet-stream;base64,H4sIAAAAAAAAAPNUSMxVSM4vBwCAGeD4CAAAAA=="""")
+      assert(read(new String(res2.bytes))("data").toString.contains("data:application/octet-stream;base64,H4sIAAAAAA"))
```

Comment:
This is interesting. So this was changed in https://github.com/com-lihaoyi/requests-scala/pull/108 from a `contains` like I have here to an exact match. However, I notice a difference between Java 8 and 17 here. With 8 you'll get

```
// "data:application/octet-stream;base64,H4sIAAAAAAAAAPNUSMxVSM4vBwCAGeD4CAAAAA=="
```
but with 17 you'll get
```
// "data:application/octet-stream;base64,H4sIAAAAAAAA//NUSMxVSM4vBwCAGeD4CAAAAA=="
```
I'm not exactly sure why, but is this potentially why it used to be a `contains` instead of an exact match to account for this? Is there something different with gzip between 8 and 17 here, and that's also why it doesn't change for the `Deflate` example down below?